### ISSUE TITLE
1.1.0 does not contain changes from #1

### ISSUE THREAD

**Issue by ssoloff** [JoinColony/jest-sandbox#2]
It appears that the `dist` folder was not updated when the 1.1.0 release was created.  Thus, the new `JestSandbox#restore()` method added in #1 is not available when that release is installed from npm.

From a fresh npm install of this module, you can observe that `dist/index.js` still has a timestamp from the 1.0.0 release:

```
$ ll node_modules/jest-sandbox/index.js node_modules/jest-sandbox/dist/index.js 
-rw-r--r--. 1 ssoloff ssoloff 1964 Apr 15 18:35 node_modules/jest-sandbox/dist/index.js
-rw-r--r--. 1 ssoloff ssoloff  546 Jun 25 13:10 node_modules/jest-sandbox/index.js
```

**Comment by chmanie** [comment]
Thanks for the feedback and sorry about that. I just published v1.1.1 to npm.

**Comment by ssoloff** [comment]
Confirmed that I can now use `restore()` with 1.1.1.  Thanks for the quick turnaround, @chmanie!
